src/mocks/RankingMock.ts

```diff
@@ -0,0 +1,68 @@
+const RankingMock = {
+  myRanking: 3,
+  myNickname: '홍길동',
+  myTime: 'PT7H30M45S',
+  Page: {
```

Comment:
다른 속성은 카멜 케이스로 사용하는데 Page만 파스칼 케이스로 되어 있네요. 오타라면 한번 확인해서 개선해보는게 좋겠습니다.


```suggestion
  page: {
```